utils/test_memcheck.supp

```diff
@@ -67,11 +67,10 @@
    Memcheck:Leak
    ...
    fun:calloc
+   ...
```

Comment:
This one seems not specific enough unless we have a real non-test bug we need to fix.

Reply:
I see, well, this one is related to a test, so I'm going to send a patch with less wildcards